AmplifyTestCommon/Helpers/Sequence+sum.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public extension Sequence where Element: SignedInteger {
+    @inlinable func sum() -> Element {
+        reduce(0, +)
+    }
+}
+
+public extension Sequence where Element: UnsignedInteger {
+    @inlinable func sum() -> Element {
+        reduce(0, +)
+    }
+}
```

Comment:
```suggestion
public extension Sequence where Element: AdditiveArithmetic {
    func sum() -> Element {
        reduce(.zero, +)
    }
}
```

If this is really only for testing purposes, the `@inlinable` is unnecessary but it also doesn't hurt.